inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -91,6 +91,24 @@ class CNNLayerCreator : public ::ngraph::AttributeVisitor {
         params[name] = std::to_string(adapter.get());
     }
 
+    void on_adapter(const std::string& name, ngraph::ValueAccessor<std::vector<std::string>>& adapter) override {
+        std::vector<std::string> data = adapter.get();
+        for (auto& str : data) {
+            std::transform(str.begin(), str.end(), str.begin(), [](unsigned char c) {
+                return std::tolower(c);
```

Comment:
Why do we need transform strings to lower case? Maybe we need a comment in the code about this.

Reply:
It is made like an  `on_adaptor ` function for a single string.  Perhaps there are attributes with capital letters, it's hard to say for sure.

Reply:
F2F discussed.